content/updates/2020-10/6-Job-and-process-tracker-multiple-improvements.md

```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Jobs and process tracker multiples improvements
```

Comment:
```suggestion
# Jobs and process tracker improvements
```
Are those improvements related to the 1st article "# Interrupt your imports and exports" (that I want to rename into "# More control over imports and exports" maybe we should put everything under this article?